lib/components/home/artist_rails/artist_rail.js

```diff
@@ -30,35 +39,88 @@ class ArtistRail extends React.Component {
 
   componentWillReceiveProps(nextProps) {
     if (nextProps.rail.results) {
-      this.setState({ artists: nextProps.rail.results })
+      const artists = nextProps.rail.results
+      artists.forEach((artist) => {
+        artist._animatedValues = {
+          opacity: new Animated.Value(1),
+          translateY: new Animated.Value(0),
+        }
+      })
+      this.setState({ artists })
     }
   }
 
-  /**
-   * Removes the followed artist from the list and re-renders.
-   */
-  handleFollowChange(artist) {
-    metaphysics(suggestedArtistQuery(artist._id))
-      .then(({ me: { suggested_artists }}) => suggested_artists[0]) // return artist or undefined
-      .catch(error => console.error(error))                         // return undefined
-      .then((suggested_artist) => {
-        const artists = this.state.artists.slice(0)                 // copy array
-        const index = artists.indexOf(artist)
-        if (suggested_artist) {
-          artists[index] = suggested_artist                         // replace card with new suggestion
-        } else {
-          artists.splice(index, 1)                                  // remove card when no suggestion or on error
-        }
-        this.setState({ artists })
-      })
+  followedArtistAnimation(followedArtist) {
+    return new Promise((resolve, reject) => {
+      const { opacity, translateY } = followedArtist._animatedValues
+      const duration = Animation.duration.followedArtist
+      const easing = Animation.easing
+      Animated.parallel([
+        Animated.timing(opacity,    { duration, easing, toValue: 0 }),
+        Animated.timing(translateY, { duration, easing, toValue: Animation.yDelta }),
+      ]).start(resolve)
+    })
+  }
+
+  suggestedArtistAnimation(suggestedArtist) {
+    return new Promise((resolve, reject) => {
```

Comment:
Do the promises here allow you to chain these methods further down below? (where you've done the `then` and `then` - lines 98 - 108)